Libraries/TestReport.psm1

```diff
@@ -392,20 +392,33 @@ Class TestSummary
 		}
 	}
 
+	[String] GetTestArea([string]$TestName) {
+		$files = Get-ChildItem ("$pwd\XML\TestCases\*.xml") -Exclude Other.xml
```

Comment:
in this way, it needs to read all test case files once for each case.
I would suggest to change the interface of UpdateTestSummaryForCase to pass CurrentTestData instead of CurrentTestName, which includes more case information